wagtail/admin/tests/test_reports_views.py

```diff
@@ -206,16 +225,19 @@ def test_filter_by_live(self):
         self.assertEqual(response.status_code, 200)
         self.assertNotContains(response, "Tentative Unpublished Event")
         self.assertContains(response, "My locked page")
+        self.assertContains(response, "Christmas")
 
         response = self.get(params={"live": "false"})
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "Tentative Unpublished Event")
         self.assertNotContains(response, "My locked page")
+        self.assertNotContains(response, "Christmas")
 
     def test_filter_by_user(self):
         response = self.get(params={"locked_by": self.user.pk})
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "Tentative Unpublished Event")
+        self.assertContains(response, "Christmas")
         self.assertNotContains(response, "My locked page")
```

Comment:
Might be worth adding something like

```py
self.assertContains(response, f'<option value="{self.user.pk}">{self.user}</option>', count=1)
```

Reply:
This was handily covered already, immediately below :-)

https://github.com/wagtail/wagtail/blob/b3b5a04ed10d7191c98c393ee07c14956cb31559/wagtail/admin/tests/test_reports_views.py#L73-L81

Reply:
Ooo nice!